vite.config.mts

```diff
@@ -114,6 +114,22 @@ export default defineConfig(({ mode }) => {
           "connect-src 'self' *.data.gouv.fr raw.githubusercontent.com dev.local:7000"
         ].join('; ')
       }
+    },
+    optimizeDeps: {
+      // commonjs dependencies must be included here
+      // otherwise Vite dev server don't process them correctly at runtime
```

Comment:
I'm not sure I understand. Is there more context somewhere, like in an issue?

Reply:
The global context is that Vite is serving ESM module only during dev and it [converts them to commonjs](https://vite.dev/guide/dep-pre-bundling.html#the-why) when it finds them.
It should find them during its [dependency scan](https://vite.dev/guide/dep-pre-bundling.html#automatic-dependency-discovery) but somehow this is not the case for these, so I followed the [documentation on how to add them when they are not discovered](https://vite.dev/guide/dep-pre-bundling.html#customizing-the-behavior).

There are [questions](https://laracasts.com/discuss/channels/vite/problem-resolving-imports-with-vite) [related](https://stackoverflow.com/questions/74845363/vue3-with-vite-cant-import-commonjs-module) to this but no real answer to understand why some package are not found

Reply:
I updated the comment to make it clear that the dependencies are from `datagouv/components-next`